src/app/Classes/OptionsBuilder.php

```diff
@@ -78,7 +78,11 @@ private function setPivotParams()
         collect(json_decode($this->request->get('pivotParams')))
             ->each(function ($param, $table) {
                 $this->query->whereHas($table, function ($query) use ($param) {
-                    $query->whereIn('id', (array) $param->id);
+                    collect($param)->each(
+                        function ($fieldValue, $fieldName) use ($query) {
+                            $query->whereIn($fieldName, (array) $fieldValue);
```

Comment:
- please rename `$fieldValue` to `$value` and `$fieldName` to `$attribute`
- also on line 79/80 I noticed that `$table` should be `$relation`, please also add this change to your commit